PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## PR Description
+
+
+## PR Checklist
+
+### Testing
+- [ ] added relevant test coverage
+- [ ] no tests needed
+
+### Docs
+- [ ] added relevant docs
+- [ ] updated relevant sections in the README.md
+- [ ] no docs needed
```

Comment:
I think we should add a section for updating the docstrings in the index.d.ts

Reply:
you got it. done!